sidebars.js

```diff
@@ -36,6 +36,11 @@ module.exports = {
         label: "ExpiringMultiParty (EMP)",
         items: ["developers/emp-deployment", "build-walkthrough/emp-parameters", "build-walkthrough/minting-etherscan",],
       },
+      {
+        type: "category",
+        label: "Deploying on Polygon",
```

Comment:
The other headers in our developer section is classified by our technology rather than the platform. I think it would be more consistent if we changed this header to `Long Short Pair (LSP)` and then the page's title to `Deploying on Polygon`. Thoughts?

Reply:
makes sense